dpanonymize/__init__.py

```diff
@@ -53,18 +67,37 @@ def anonymize(self, **kwargs) -> None:
                     self.file_path,
                     self.general_path)
         else:
+            if module == None:
+                raise ModuleDetectionError(
+                        f'Module is not detected: (dtype={self.dtype})')
             module.remove_pii(self.file_path, self.general_path)
 
     def __repr__(self):
-        return f"<{self.file_path.name}>"
+        return f"<{self.file_path.name} {self.dtype}"
+
+
+class ModuleDetectionError(Exception):
+    pass
 
 
 class FileInPhoenix(FileInPhoenixBIDS):
-    '''NON-BIDS PHOENIX file class used to grab file information'''
-    def __init__(self, file_path):
+    '''NON-BIDS PHOENIX file class used to grab file information
```

Comment:
If `NON-BIDS`, why does the argument say `FileInPhoenixBIDS`?

Reply:
It's just inheriting from `FileInPhoenixBIDS` class. `FileInPhoenix` is for non-bids. Thanks for double checking.